src/Auth/BaseAuthenticate.php

```diff
@@ -168,7 +171,7 @@ protected function _query($username)
      * @throws \RuntimeException If password hasher class not found or
      *   it does not extend AbstractPasswordHasher
      */
-    public function passwordHasher()
+    public function passwordHasher(): AbstractPasswordHasher
```

Comment:
Same here

Reply:
So the annotation is simply wrong here?

Reply:
I didnt say wrong. I basically meant "not ideal". Never mind, though, in this case.